ffi/capi/src/pluralrules.rs

```diff
@@ -53,6 +54,44 @@ pub extern "C" fn icu4x_plural_rules_create(
     }
 }
 
+#[repr(C)]
+#[derive(Default)]
+/// This is the result returned by [`icu4x_plural_operands_create()`]
+pub struct ICU4XCreatePluralOperandsResult {
+    /// Will default initialized if `success` is [`false`]
+    pub operands: ICU4XPluralOperands,
+    /// Currently just a boolean, but we might add a proper error enum as necessary
+    pub success: bool,
+}
+
+#[no_mangle]
+/// FFI version of [`PluralOperands::from_str()`]. See its docs for more details.
+///
+/// # Safety
+/// `number` and `len` should point to a valid ASCII string of length `len`.
+/// The string `number` should be able to parse as a floating-point value.
+///
+/// It does not need to be be null terminated, and `len` should not include a null
+/// terminator (this will just cause the function to panic, and is not a safety requirement).
+pub unsafe extern "C" fn icu4x_plural_operands_create(
+    number: *const u8,
+    len: usize,
+) -> ICU4XCreatePluralOperandsResult {
+    // cheap as long as there are no variants
+    let bytes = slice::from_raw_parts(number, len);
+
+    // todo: return errors
```

Comment:
issue: we should probably handle this before landing

Reply:
Ah sorry, I think that I had a misunderstanding here. 

I looked at the `icu4x_create_locale` as an example, which has this same `todo` item. 
https://github.com/unicode-org/icu4x/blob/a0d4cf41951316c7f1c8733e20e8577fcd7e3f51/ffi/capi/src/locale.rs#L24

I took that to mean "return real error types instead of a boolean", but I see now that `icu4x_create_locale` just returns a `Locale` object regardless, and not a `Result` type with a `success` boolean. 

I believe that I am handling the errors to the extent that we currently handle them in this code, which is a boolean flag. 

I will remove the comment, since I believe it is not relevant, but I believe that this code should be sufficient unless I am misunderstanding you.

Reply:
Actually, let me just do that anyway. I prefer to be explicit about things.